react-sizeme.d.ts

```diff
@@ -3,29 +3,42 @@ import { Component, ComponentType, ReactNode, ReactElement } from 'react'
 type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
 
 export interface SizeMeProps {
-  size: {
-    width: number | null
-    height: number | null
+  readonly size: {
+    readonly width: number | null
+    readonly height: number | null
   }
 }
 
 export interface SizeMeOptions {
-  monitorWidth?: boolean
-  monitorHeight?: boolean
-  monitorPosition?: boolean
-  refreshRate?: number
-  refreshMode?: 'throttle' | 'debounce'
-  noPlaceholder?: boolean
+  readonly monitorWidth?: boolean
```

Comment:
Not sure if there is any value marking these as read-only as they won't get exposed to the user. These configuration values are inputs.